src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3833,15 +3819,15 @@ public override BoundNode VisitConditionalAccess(BoundConditionalAccess node)
                 (alternativeLValue, alternativeRValue) = visitConditionalRefOperand(alternativeState, originalAlternative);
                 Join(ref this.State, ref consequenceState);
 
-                TypeSymbol refResultType = node.Type.SetUnknownNullabilityForReferenceTypes();
+                TypeSymbol refResultType = node.Type!.SetUnknownNullabilityForReferenceTypes();
```

Comment:
Seems like it should be an assertion. Same a few lines below

Reply:
Both will result in exceptions if the type is null.

---
In reply to: [463273842](https://github.com/dotnet/roslyn/pull/45640#discussion_r463273842) [](ancestors = 463273842)